modules/rosa-configure.adoc

```diff
@@ -0,0 +1,271 @@
+:system-module-type: REFERENCE
+// Module included in the following assemblies:
+//
+// * cli_reference/rosa_cli/rosa-get-started-cli.adoc
+
+[id="rosa-configure_{context}"]
+= Configuring `rosa`
+
+[role="_abstract"]
+Use the following commands to configure the `rosa` CLI.
+
+[id="rosa-login_{context}"]
+== login
+
+Logs in to your Red Hat account, saving the credentials to the `rosa` configuration file. You must provide a token when logging in. You can copy your token by from link:https://cloud.redhat.com/openshift/token/rosa[the {product-title} token page].
+
+`rosa` looks for a token in the following priority order:
```

Comment:
ISG suggests not starting out a sentence with a lowercase word. Perhaps you could rewrite to say something like, "The `rosa` CLI looks for..."

Reply:
Done.